src/software/ai/hl/stp/play/tactic_assignment_test.cpp

```diff
@@ -77,14 +77,16 @@ TEST_F(STPTacticAssignmentTest,
     move_tactic_2->updateControlParams(Point(1, 0), Angle::zero(), 0,
                                        TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT);
 
-    ConstTacticVector tactics = {move_tactic_1, move_tactic_2};
+    TacticVector tactics = {move_tactic_1, move_tactic_2};
 
-    auto asst = stp.assignRobotsToTactics({tactics}, world, true);
+    auto robot_navigation_obstacle_config = ai_config.robot_navigation_obstacle_config();
+    GlobalPathPlannerFactory path_planner_factory(robot_navigation_obstacle_config, world.field());
```

Comment:
since you are using `path_planner_factory` in every test, you can move variable into the TacticAssignmentTest class so that you don't have to keep recreating this variable in every test